src/commands.rs

```diff
@@ -229,15 +229,14 @@ impl<'a, V: SelectValue> KeyValue<'a, V> {
                 })])
             } else {
                 // Adding somewhere in existing object, use jsonpath_lib::replace_with
+                let p = parsed_static_path
+                    .static_path_elements
+                    .iter()
+                    .map(|e| e.to_string())
+                    .collect::<Vec<String>>()
```

Comment:
Can use Inferred type
```suggestion
                    .collect::<Vec<_>>()
```